docs/admin/runai-setup/self-hosted/k8s/upgrade.md

```diff
@@ -79,30 +86,59 @@ kubectl patch pvc -n runai-backend pvc-postgresql  -p '{"metadata": {"annotation
 
 Delete the ingress object which will be recreated by the control plane upgrade
 
-```
+``` bash
 kubectl delete ing -n runai-backend runai-backend-ingress
 ```
-#### Installation Customization
+#### Installation customization
 
 The Run:ai control-plane installation has been rewritten and is no longer using a _backend values file_. Instead, to customize the installation use standard `--set` flags. If you have previously customized the installation, you must now extract these customizations and add them as `--set` flag to the helm installation:
 
 * Find previous customizations to the control plane if such exist. Run:ai provides a utility for that here `https://raw.githubusercontent.com/run-ai/docs/v2.13/install/backend/cp-helm-vals-diff.sh`. For information on how to use this utility please contact Run:ai customer support. 
 * Search for the customizations you found in the [optional configurations](./backend.md#optional-additional-configurations) table and add them in the new format. 
 
-### Upgrade Control Plane
 
+## Upgrade Control Plane
+
+### Upgrade from version 2.13, or later
+
+=== "Connected"
+
+    ``` bash
+    helm get values runai-backend -n runai-backend > runai_control_plane_values.yaml
+    helm upgrade runai-backend -n runai-backend runai-backend/control-plane -f runai_control_plane_values.yaml --reset-then-reuse-values
+    ```
+=== "Airgapped"
+
+    ``` bash
+    helm get values runai-backend -n runai-backend > runai_control_plane_values.yaml
+    helm upgrade runai-backend control-plane-<NEW-VERSION>.tgz -n runai-backend  -f runai_control_plane_values.yaml --reset-then-reuse-values
+    ```
+
+### Upgrade from version 2.7, 2.8, 2.9, or 2.11
+    
 * Create a `tls secret` as described in the [control plane installation](backend.md). 
 * Upgrade the control plane as described in the [control plane installation](backend.md). During the upgrade, you must tell the installation __not__ to create the two PVCs:
 
-```
-helm upgrade -i runai-backend -n runai-backend runai-backend/control-plane \
+=== "Connected"
+
+    ``` bash
+    helm upgrade -i runai-backend -n runai-backend runai-backend/control-plane \
     --set global.domain=<DOMAIN> \
     --set postgresql.primary.persistence.existingClaim=pvc-postgresql \ 
     --set thanos.receive.persistence.existingClaim=pvc-thanos-receive 
-```
+    ```
+ 
+=== "Airgapped"
+
+    ``` bash
+    helm upgrade -i runai-backend control-plane-<NEW-VERSION>.tgz -n runai-backend runai-backend/control-plane \
```

Comment:
this line should be changed 
helm upgrade -i runai-backend control-plane-<NEW-VERSION>.tgz -n runai-backend  \